### DISCUSSION TITLE
VS GUI question

### DISCUSSION THREAD

**Discussion by siblount** [d2phap/ImageGlass#1284] [Q&A] [answered]
Hi,

I saw in the Wiki that the GUI is designed at 144 DPI and I should run Visual Studio at 150% magnification. I was just wondering if that's still a requirement since it's a bit dated.

Thanks!

**Comment by d2phap** [comment]
My VS 2022 designer is at 140 DPI. I think it's not so important.

**Comment by fire-eggs** [comment]
In the past, if I made any GUI changes at 100%, the resulting layout would be very different from what was in Github. Mostly positioning, some sizing.

**Comment by siblount** [comment, marked as answer]
Ok, so I have it at 125% scaling. Here is what it looks like for me: <image src="https://i.ibb.co/xg5FTP4/image.png">

Hopefully that is what is supposed to look like.
